### ISSUE TITLE
How to list gcr.io/distroless/python3-debian11 tags?

### ISSUE THREAD

**Issue by toddysm** [GoogleContainerTools/distroless#1051]
For the rest of the distroless images there is a page that lists the available tags. However, for the python one the page doesn't contain tag list. Using the standard [distribution v2 API](https://docs.docker.com/registry/spec/api/#listing-image-tags) `curl https://gcr.io/v2/distroless/python3-debian11/tags/list` returns different result that the one in the spec.

**Comment by cpanato** [comment]
if you use crane you can see the tags

```
$ crane ls  gcr.io/distroless/python3-debian11
debug
debug-amd64
debug-arm64
debug-nonroot
debug-nonroot-amd64
debug-nonroot-arm64
latest
latest-amd64
latest-arm64
nonroot
nonroot-amd64
nonroot-arm64
....
```

**Comment by loosebazooka** [comment]
Yeah this should probably be documented somewhere, where are you finding the tags for non-python images?
Also we should untag unsupported images.

**Comment by omBratteng** [comment]
I usually check the Google Cloud page, https://console.cloud.google.com/gcr/images/distroless/GLOBAL/python3-debian11.
You'd be re-directed there if you navigate to `gcr.io/distroless/python3-debian11` in your browser.
